gulpfile.js

```diff
@@ -57,6 +57,8 @@ const configApp = {
         'public/scripts/views/forms/move-down.js',
         'public/scripts/views/forms/move-up.js',
         'public/scripts/views/forms/nav.js',
+        'public/scripts/views/forms/oauth-apple.js',
```

Comment:
The Apple file was deprecated in favor of oauth-custom.js which should cover most use-cases.